skflow/estimators/base.py

```diff
@@ -197,6 +198,12 @@ def fit(self, X, y, logdir=None):
         self._data_feeder = setup_train_data_feeder(X, y,
                                                     self.n_classes,
                                                     self.batch_size)
+
+        if monitor is None:
+            self._monitor = monitors.BaseMonitor()
```

Comment:
just a though may be for subsequent PR is to have `montiors.default_monitor()` function that returns this or another class. For example, this would allow to be able override default with `set_default_monitor()` or easily change which one is default if this line will be more then in one place.